Disable this test because it attempts to do a DNS lookup, which fails
in a chroot.

--- myserver-0.10/tests/test_socket.cpp	2010-10-19 00:12:59.000000000 +0200
+++ myserver-0.10/tests/test_socket.cpp	2010-10-19 00:13:07.000000000 +0200
@@ -56,7 +56,6 @@ class TestSocket : public CppUnit::TestF
 
   CPPUNIT_TEST (testGethostname);
   CPPUNIT_TEST (testRecv);
-  CPPUNIT_TEST (testGetLocalIPsList);
 
   CPPUNIT_TEST_SUITE_END ();
